lib/workers/repository/process/lookup/index.ts

```diff
@@ -200,6 +205,8 @@ export async function lookupUpdates(
         return res;
       }
     }
+    // Reapply package rules in case we missed something from sourceUrl
+    config = applyPackageRules(config);
```

Comment:
```suggestion
    config = applyPackageRules({ ...config, sourceUrl: res.sourceUrl });
```

Could this line fix it by itself?

Reply:
Yeah, I think this looks better.